authors/apps/articles/tests.py

```diff
@@ -334,3 +339,119 @@ def test_correct_time_less_than_2_minutes(self):
     def test_correct_time_less_than_1_minute(self):
         time = Article.article_reading_time(self.body * 20)
         self.assertEqual(time, 'Less than a minute')
+class BookmarkModelTest(TestCase):
```

Comment:
Please add two blank lines before beginning this class.

Reply:
Thanks, worked on it.